src/TToolBar.cpp

```diff
@@ -59,9 +59,7 @@ TToolBar::TToolBar(TAction* pA, const QString& name, QWidget* pW)
 
 void TToolBar::resizeEvent(QResizeEvent* e)
 {
-    if (!mudlet::self()->mIsLoadingLayout) {
```

Comment:
Yes, but that doesn't matter because the layout loader calls `Host::commitLayoutUpdates(true)` when it's done, which flushes any layout change notifications that might have arrived while it works.

Besides, as far as I can determine loading a layout is a synchronous action that doesn't call `qApp.processEvents`, so AFAICT this cannot happen in the first place. But I'll verify that before doing even more surgery.